Orso.Arpa.Api/Controllers/AuthController.cs

```diff
@@ -31,9 +31,8 @@ public AuthController(IAuthService authService)
         [HttpPost("login")]
         [AllowAnonymous]
         [ProducesResponseType(StatusCodes.Status200OK)]
-        [ProducesResponseType(StatusCodes.Status400BadRequest)]
-        [ProducesResponseType(StatusCodes.Status401Unauthorized)]
-        [ProducesDefaultResponseType]
+        [ProducesResponseType(typeof(ErrorMessage), StatusCodes.Status400BadRequest)]
+        [ProducesResponseType(typeof(ErrorMessage), StatusCodes.Status401Unauthorized)]
```

Comment:
there are other _error_ responses in the whole API specification of the backend that lack the `typeof(ErrorMessage),` parameter. Please tidy up mid-term.

Reply:
@euphi967 Thanks for reviewing. I checked all controllers, but did not find any other incomplete error response types. Can you give me a clue where exactly you noticed this? However, I found and fixed another bug (see commit). Can you please check it and approve it again? Thank you!

Reply:
fair point :-). I came from reviewing the `swagger.json` for _schema/$ref_ and I detected several 4xx responses with no _schema/$ref_ tag. Looking again showed, that only the 403 responses are lacking the _schema/$ref_.

Reply:
how will we deal with those 403s? No need to specify _schema/$ref_ ?